README.md

```diff
@@ -484,6 +484,7 @@ Hurray! You successfully made a contribution! 🎉
   - [Shanmuganathanb](https://github.com/Shanmuganathanb)
   - [Shatakshi Gupta](https://github.com/Shatakshi0805)
   - [Shaun Fitzgibbon](https://github.com/Sfitzg)
+  - [Shaikh Mohammad Danish](https://github.com/SMD-1)
```

Comment:
Please add your name in line 483.

Reply:
sorry @SMD-1 didn't realise it was meant to go on line 483